### DISCUSSION TITLE
key binding for "next test" button

### DISCUSSION THREAD

**Discussion by AkashKishore-Mukhia** [monkeytypegame/monkeytype#1929] [Ideas] [answered]
I think it will be very convenient to have a key shortcut for the "next test" and other buttons below the result window.

**Comment by D-T-666** [comment, marked as answer]
`tab` + `enter` does that
![image](https://user-images.githubusercontent.com/35934791/135619020-cd230465-ab61-4a43-9ba1-bf4733e8f254.png)

**Reply by Miodec** [reply]
Tab + enter or just tab with quick tab mode enabled.
